frontend/src/features/manageProfile/components/dashboard/ManageProfilesDashboard.tsx

```diff
@@ -45,8 +47,11 @@ export const ManageProfilesDashboard = React.memo(() => {
 
   const navigate = useNavigate();
   const { userRoles } = useContext(OnRouteBCContext);
+  const { user } = useAuth();
   const populatedUserRoles = getDefaultRequiredVal([], userRoles);
-  const isBCeIDAdmin = isBCeIDOrgAdmin(populatedUserRoles);
+  const isBCeIDAdmin =
```

Comment:
not a biggie but it seems like you're duplicating the `isIDIR(blah)` stuff. What about above line 52 creating:
```
const isIDIR = isIDIR(user?.profile?.identity_provider as string);
```
then referencing that from line 54 and line 82?